.travis.yml

```diff
@@ -1,55 +1,141 @@
+sudo: false
 language: android
 
-android:
-  components:
-    - tools
-    - platform-tools
-    - build-tools-27.0.0
-    - android-16
-    - android-19
-    - android-21
-    - android-27
-    - extra-android-m2repository
-    - sys-img-armeabi-v7a-android-16
-    - sys-img-armeabi-v7a-android-19
-    - sys-img-armeabi-v7a-android-21
+env:
+- ADB_INSTALL_TIMEOUT=5 # increase from default of 2 mins
+
+before_install:
+- bundle install
+- export TERM=dumb
+- curl -L https://dl.google.com/android/repository/android-ndk-r16b-linux-x86_64.zip -O
```

Comment:
> A script exists for this in bugsnag-android-ndk, it'd be good to update that instead of duplicating here, as that could also be useful for anyone testing the example app out.

The script in `bugsnag-android-ndk` just tests whichever NDK version it can get from `sdkmanager`, which is only the latest version. This is specifically testing r16b.

> I'm also a bit concerned that testing the NDK + SDK together could push us over the time limit for Travis CI jobs, as we are already quite close to the limit.

True, though in that case we should pay for more time.

Reply:
The script in bugsnag-android-ndk I'm thinking of specifically grabs r16b: https://github.com/bugsnag/bugsnag-android-ndk/blob/master/install_ndk.sh